### PR TITLE
Correct spelling and grammar improvements

### PR DESCRIPTION
**Pull request by savvar9991**
1. **Spelling Corrections**:

  - `useable` → `usable` (Correct spelling of the word)
   - `expensive_calulation` → `expensive_calculation` (The word "calulation" is incorrectly spelled, and the correct spelling is "calculation" from the verb "to calculate")

2. **Grammar Improvement**:

- `be at conflict` → `be in conflict` (The preposition "at" is replaced with "in" because the correct expression is "to be in conflict with," which always uses the preposition "in")

### CODE REVIEW

File: doc/build-osx.md [hunk 1]
```diff
@@ -71,7 +71,7 @@ It is not necessary to build wallet functionality to run `bitcoind` or  `bitcoin
 
 `sqlite` is required to support for descriptor wallets.
 
-macOS ships with a useable `sqlite` package, meaning you don't need to
+macOS ships with a usable `sqlite` package, meaning you don't need to
```
Comment (l0rinc) [inline doc/build-osx.md:74]: https://www.vocabulary.com/dictionary/useable

**Review by jonatack — COMMENTED** [review]
ACK, modulo:

- unnecessary change per https://github.com/bitcoin/bitcoin/pull/31517/files#r1888478161
- please squash your commits as described in https://github.com/bitcoin/bitcoin/blob/master/CONTRIBUTING.md#squashing-commits
- preferably add the `doc: ` prefix to the PR title

File: doc/translation_strings_policy.md
Comment (Redlund99) [inline doc/translation_strings_policy.md:1]: Both "at conflict with" and "in conflict with" are grammatically correct.

**Comment by glozow** [comment]
Thanks for your interest in contributing! However, as we have hundreds of pull requests, I am closing this to focus review on the others. (See [contributing guidelines](https://github.com/bitcoin/bitcoin/blob/master/CONTRIBUTING.md#refactoring)).

**Pull request closed without merging**